NARFO_BE/NARFO_BE/Controllers/MembersController.cs

```diff
@@ -2,121 +2,133 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Cors;
 using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.EntityFrameworkCore;
 using NARFO_BE.Models;
 
 namespace NARFO_BE.Controllers
 {
-    [Route("api/[controller]")]
-    [ApiController]
+    [Route("Member")]
+    [EnableCors("MyPolicy")]
     public class MembersController : ControllerBase
     {
         private readonly narfoContext _context;
-        List<Members> members = new List<Members>();
+        List<_Member> members = new List<_Member>();
+        public UserManager<_Member> UserManager { get; private set; }
 
         public MembersController(narfoContext context)
         {
             _context = context;
             if (!_context.Members.Any())
             {
-                Members newMember = new Members();
-                newMember.LastName = "admin";
+                _Member newMember = new _Member();
+                newMember.SURNAME = "admin";
                 newMember.Username = "admin";
-                newMember.FirstName = "admin";
-                newMember.Password = "admin";
+                newMember.Firstname = "admin";
+                newMember.Email = "admin@admin.com";
+               
                 _context.Members.AddAsync(newMember);
                 _context.SaveChanges();
             }
         }
-        
-        public void ListofMembers(List<Members> members)
-        {
-            this.members = members;
-        }
-        
 
-        // GET: api/Members
-        [HttpGet]
-        public async Task<ActionResult<IEnumerable<Members>>> GetMembers()
-        {
-            return await _context.Members.ToListAsync();
-        }
-        public IEnumerable<Members> GetAllMembers()
+        public void ListofMembers(List<_Member> members)
```

Comment:
This is a set function right? List of Members doesn't really describe what this function does.

Reply:
the function turned to be useless.. we'll delete it thank you for your view ( :

Reply:
Ok